x-pack/plugins/observability/public/locators/slo_edit.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { setStateToKbnUrl } from '@kbn/kibana-utils-plugin/public';
+import type { RecursivePartial } from '@elastic/charts';
+import type { SerializableRecord } from '@kbn/utility-types';
+import type { LocatorDefinition } from '@kbn/share-plugin/public';
+import type { SLOWithSummaryResponse } from '@kbn/slo-schema';
+import { sloEditLocatorID } from '../../common';
+import { SLO_CREATE_PATH, SLOS_PATH } from '../routes/paths';
+
+export type SloEditParams = RecursivePartial<SLOWithSummaryResponse>;
```

Comment:
I think this should be a `RecursivePartial<CreateSLOForm>`

Reply:
You are right!